### PR TITLE
Refactoring the "My Uploads" tab

### PR DESCRIPTION
**Pull request by jettlin**
Redid the "My Uploads" tab to have a table view, ensuring previous functionality still exists, and added in views for each video pulled from the player endpoint.

### CODE REVIEW

File: synq-video.php [hunk 1]
```diff
@@ -156,6 +202,26 @@ function uploadspage(){
...
+function formatTime($seconds) {
+  $hours = floor($seconds / 3600);
+  $minutes = floor(($seconds - ( $hours * 3600 ))/ 60);
+  $seconds = $seconds % 60;
+
+  return ( $hours > 0 ? $hours . ":" : '' ) . ( $minutes > 0 ? $minutes : '0' ) . ":" . ( $seconds >= 10 ? $seconds : "0" . $seconds );
+}
+
+function getViews() {
+  $response = file_get_contents("http://player.synq.fm/events?api_key=" . get_option('synq_api_key'), $false, $context);
```
Comment (batmany13) [inline synq-video.php]: we should use https://player.synq.fm if we can.  If it doesn't work, we need to make sure https is enabled

**Comment by batmany13** [comment]
this fixes #14

**Comment by jettlin** [comment]
@batmany13 Updated to use HTTPS.  Can you verify the changes?

**Comment by gsemino** [comment]
Changes look good, tested uploading some videos, the old functionality still works with the new changes, such as the table views in both the 'Uploaded Videos' section and 'upload a video' section in a page or post. The copy to clipboard function works and when clicking a thumbnail for a video a new page tab opens so you can view the video. The new tab names are also present.

**Pull request merged**